modules/tinymce/src/core/main/ts/api/NotificationManager.ts

```diff
@@ -155,10 +155,14 @@ const NotificationManager = (editor: Editor): NotificationManager => {
       });
     });
 
-    editor.addShortcut('alt+F12', 'Focus to notification', () =>
-      getTopNotification()
-        .map((notificationApi) => SugarElement.fromDom(notificationApi.getEl()))
-        .each((elm) => Focus.focus(elm)));
+    editor.on('keydown', (e) => {
+      if ((e.altKey || e.metaKey) && e.key.toLowerCase() === 'f12') {
```

Comment:
Not sure we should allow both meta and alt on both Mac OS and Windows/Linux?
Are we sure meta is even right on any platform the old key binding seems to be alt only?

Reply:
`meta` is `command` on macOS. So no, we should not allow meta.

Reply:
yeah the `meta` key was a mistake, sorry about that